docs/reporting/tutorial-arcgis-notebook.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Run a report from an ArcGIS Notebook
+description: Geocortex Reporting - ArcGIS Notebook
+---
+
+You can use the **Geocortex Reporting Client for Python** package to run reports from ArcGIS Notebooks.
+
+## Prerequisites
+
+-   ArcGIS Online or ArcGIS Enterprise 10.7+
+-   An ArcGIS user with permission to create and edit ArcGIS Notebooks
+-   An existing Geocortex Report that accepts a `FeatureIds` parameter
+
+## Author the Notebook
+
+1. Sign into ArcGIS Online or ArcGIS Enterprise as a Notebook Author user
+2. Create a new Notebook
+3. Add a cell with the following script:
+
+```py
+!pip install geocortex-reporting-client
+```
+
+4. Run the cell to install the `geocortex-reporting-client` package
+5. Add a cell with the following script:
+
+```py
+from geocortex.reporting.client import run
+
+report = await run("itemid", FeatureIds=[1, 2, 3])
```

Comment:
Do you think it would be useful to show the token usage from their API?

Reply:
Yes, it totally would. I've added #80 to deal with that separately.